### PR TITLE
Update descriptions

### PR DESCRIPTION
**Pull request by JStickler**
**What this PR does / why we need it**:

Updates the `description` field in the metadata to support the Docs Landing Page project

### CODE REVIEW

File: docs/sources/operations/autoscaling_queriers.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Autoscaling Loki queriers
 menuTitle: Autoscaling queriers
-description: Kubernetes deployments of a microservices mode Loki cluster can use KEDA to autoscale the quantity of queriers.
+description: Describes how to use KEDA to autoscale the quantity of queriers for a midrosevices mode Kubernetes deployment.
```
Comment (MichelHollands) [inline docs/sources/operations/autoscaling_queriers.md:4]:
```suggestion
description: Describes how to use KEDA to autoscale the quantity of queriers for a microsevices mode Kubernetes deployment.
```

File: docs/sources/operations/overrides-exporter.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Overrides exporter
 menuTitle:  
-description: Overrides Exporter
+description: Describes how the Overrides Exporter module exposes tenant limits as Promtheus metrics.
```
Comment (MichelHollands) [inline docs/sources/operations/overrides-exporter.md:4]:
```suggestion
description: Describes how the Overrides Exporter module exposes tenant limits as Prometheus metrics.
```

File: docs/sources/query/metric_queries.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Metric queries 
 menuTItle:  
-description: Metric queries extend log queries by applying a function to log query results. This powerful feature creates metrics from logs.
+description: Provides an overview of how metric queries are constructed and parsed.  Metric queries extend log queries by applying a function to log query results.
```
Comment (MichelHollands) [inline docs/sources/query/metric_queries.md:4]:
```suggestion
description: Provides an overview of how metric queries are constructed and parsed. Metric queries extend log queries by applying a function to log query results.
```

**Review by MichelHollands — APPROVED** [review]
There are a few typos but otherwise LGTM

**Comment by JStickler** [comment]
@MichelHollands thanks for catching my typos!

**Pull request merged**